src/main/java/graphql/collect/ImmutableKit.java

```diff
@@ -61,25 +61,26 @@ public static <K, V> ImmutableMap<K, V> mergeMaps(Map<K, V> m1, Map<K, V> m2) {
     }
 
     public static <T> ImmutableList<T> concatLists(List<T> l1, List<T> l2) {
-        return ImmutableList.<T>builder().addAll(l1).addAll(l2).build();
+        //noinspection UnstableApiUsage
+        return ImmutableList.<T>builderWithExpectedSize(l1.size() + l2.size()).addAll(l1).addAll(l2).build();
     }
 
     /**
      * This is more efficient than `c.stream().map().collect()` because it does not create the intermediate objects needed
      * for the flexible style.  Benchmarking has shown this to outperform `stream()`.
      *
-     * @param iterable the iterable to map
+     * @param iterable the collection to map
      * @param mapper   the mapper function
      * @param <T>      for two
      * @param <R>      for result
      *
      * @return a map immutable list of results
      */
-    public static <T, R> ImmutableList<R> map(Iterable<? extends T> iterable, Function<? super T, ? extends R> mapper) {
+    public static <T, R> ImmutableList<R> map(Collection<? extends T> iterable, Function<? super T, ? extends R> mapper) {
```

Comment:
Maybe you noticed, but it's still called `iterable` as a param

Does not matter much but iterator is not the greatest denominator of the parameter anymore

Reply:
@jord1e fixed, thank you again!